src/assets/index.js

```diff
@@ -1,10 +1,14 @@
-export { default as ourServiceImage1} from "./images/our-service-1.png"
-export { default as ourServiceImage2} from "./images/our-service-2.png"
-export { default as ourServiceImage3} from "./images/our-service-3.png"
-export { default as rightArrowIcon} from "./images/right-arrow-icon.png"
+export { default as workImg1 } from './homepage/our-work-1.png';
+export { default as workImg2 } from './homepage/our-work-2.png';
+export { default as workImg3 } from './homepage/our-work-3.png';
+export { default as workImg4 } from './homepage/our-work-4.png';
+export { default as ourServiceImage1 } from './images/our-service-1.png';
+export { default as ourServiceImage2 } from './images/our-service-2.png';
+export { default as ourServiceImage3 } from './images/our-service-3.png';
+export { default as rightArrowIcon } from './images/right-arrow-icon.png';
 export { default as instagramIcon } from './icons/icon-instagram.svg';
 export { default as linkedinIcon } from './icons/icon-linkedin.svg';
 export { default as twitterIcon } from './icons/icon-twitter.svg';
 export { default as brandLogo } from './icons/brand-logo.svg';
+export { default as forwardArrow } from './icons/forward-arrow.svg';
```

Comment:
is forwardArrow and rightArrow not same?

Reply:
forwardArrow is arrow with blue background and rightArrow is only an arrow